pages/docs/1_query/1_getting_started/6_setup_endpoint.md

```diff
@@ -90,7 +90,120 @@ $ comunica-sparql-http https://fragments.dbpedia.org/2016-04/en \
+When using a context, the sources need to be provided within said context.
+</div>
+
+## 8. Emitting VoID metadata
+
+<div class="note">
+Experimental feature: can be very slow to generate the descriptions for big datasets. But only the first time and after update queries, since caching is implemented and invalidated after updating.
+</div>
+
+Using `--emitVoid` you can make your endpoint include a [VoID](https://www.w3.org/TR/void/) description in its service description:
+
+```bash
+$ comunica-sparql-http -c config.json --emitVoid
+```
+
+The metadata can be found in the service description at http://localhost:3000/sparql.
+
+You can also add [Dublin Core Metadata Terms](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/) (general metadata) using a `dcterms` field in the context:
+```json
+{
+  "sources": [
+    "https://www.rubensworks.net/",
+    "https://ruben.verborgh.org/profile/"
+  ],
+  "dcterms": {
+    "title": "comunica SPARQL endpoint",
+    "creator": "me"
+  }
+}
+```
+
+The service description with VoID included for the above context will look like this:
+```ttl
+</sparql> a <http://www.w3.org/ns/sparql-service-description#Service>;
+    <http://www.w3.org/ns/sparql-service-description#endpoint> </sparql>;
+    <http://www.w3.org/ns/sparql-service-description#url> </sparql>;
+    <http://www.w3.org/ns/sparql-service-description#feature> <http://www.w3.org/ns/sparql-service-description#BasicFederatedQuery>;
+    <http://www.w3.org/ns/sparql-service-description#supportedLanguage> <http://www.w3.org/ns/sparql-service-description#SPARQL10Query>, <http://www.w3.org/ns/sparql-service-description#SPARQL11Query>;
+    <http://www.w3.org/ns/sparql-service-description#resultFormat> <https://comunica.linkeddatafragments.org/#results_stats>, <http://www.w3.org/ns/formats/SPARQL_Results_JSON>, <http://www.w3.org/ns/formats/Shaclc>, <http://www.w3.org/ns/formats/ShaclcExtended>, <http://www.w3.org/ns/formats/JSON-LD>, <http://www.w3.org/ns/formats/N-Quads>, <http://www.w3.org/ns/formats/N-Triples>, <http://www.w3.org/ns/formats/TriG>, <http://www.w3.org/ns/formats/N3>, <http://www.w3.org/ns/formats/Turtle>, <https://comunica.linkeddatafragments.org/#results_tree>, <https://comunica.linkeddatafragments.org/#results_table>, <http://www.w3.org/ns/formats/SPARQL_Results_XML>, <http://www.w3.org/ns/formats/SPARQL_Results_TSV>, <http://www.w3.org/ns/formats/SPARQL_Results_CSV>, <https://comunica.linkeddatafragments.org/#results_simple>, <https://comunica.linkeddatafragments.org/#results_JSON>;
+    <http://www.w3.org/ns/sparql-service-description#defaultDataset> _:defaultDataset.
+_:defaultDataset a <http://www.w3.org/ns/sparql-service-description#Dataset>, <http://rdfs.org/ns/void#Dataset>;
+    <http://rdfs.org/ns/void#sparqlEndpoint> </sparql>;
+    <http://rdfs.org/ns/void#vocabulary> <http://purl.org/dc/terms/>;
+    <http://purl.org/dc/terms/title> <comunica SPARQL endpoint>;
+    <http://purl.org/dc/terms/creator> <me>;
```

Comment:
Maybe the example can be updated to use a valid IRI, because `me` is not a valid IRI.